mail-filter/mimedefang/files/mimedefang.init

```diff
@@ -0,0 +1,104 @@
+#!/sbin/openrc-run
+#
+# Generic start/stop script for MIMEDefang.  Should work on most
+# flavors of UNIX.
+
+RETVAL=0
+prog='mimedefang'
+PROGDIR=/usr/bin
+SPOOLDIR='/var/spool/MIMEDefang'
+PID="${SPOOLDIR}/${prog}.pid"
+MXPID="${SPOOLDIR}/${prog}-multiplexor.pid"
+LOCK="${SPOOLDIR}/${prog}.lock"
+MXLOCK="${SPOOLDIR}/${prog}-multiplexor.lock"
+
+# Make sure required vars are set
+SOCKET=${SOCKET:=${SPOOLDIR}/${prog}.sock}
+MX_SOCKET=${MX_SOCKET:=${SPOOLDIR}/${prog}-multiplexor.sock}
+
+depend() {
+	use antivirus
+}
+
+start() {
+	if test -r ${PID} ; then
```

Comment:
This is reinventing the wheel. OpenRC is supposed to know whether the service is running. You should also look through OpenRC manuals, as there is some new fancy program tracking magic these days, and new format of config files that use less methods and more declarations.

Reply:
Changed init script. Please see new commit.